### PR TITLE
DOC: Add documentation for round and round_ aliases

### PR DESCRIPTION
**Pull request by aparnabushan**
Added docstring for round and round_ as aliases that both refer to around() 
This is a fix to the bug - https://github.com/numpy/numpy/issues/19717

### CODE REVIEW

File: numpy/core/fromnumeric.py [hunk 1]
```diff
@@ -3276,7 +3276,8 @@ def around(a, decimals=0, out=None):
     See Also
     --------
     ndarray.round : equivalent method
-
+    numpy.round: equivalent method
```
Comment (Mukulikaa) [inline numpy/core/fromnumeric.py:3279]:
```suggestion
```
`numpy.round` does not exist in the documentation so we cannot add it to the `See Also` for now. I think a good option would be to write in the `Notes` that `np.round` is often used as an alias for `np.around`. That would potentially clear up any confusion as to why `np.round` is used in the examples in the Notes section.

File: numpy/core/fromnumeric.py [hunk 2]
```diff
@@ -3276,7 +3276,8 @@ def around(a, decimals=0, out=None):
...
+    numpy.round_: equivalent method
```
Comment (Mukulikaa) [inline numpy/core/fromnumeric.py:3280]:
```suggestion
    round_ : equivalent method
```
This is a nit just to maintain consistency with the rest of the documentation.

**Comment by InessaPawson** [comment]
@aparnabushan It would be nice to fix #19717. Do you think you could finish this PR?

**Comment by InessaPawson** [comment]
I’ve opened a new PR (https://github.com/numpy/numpy/pull/21853) to continue the work started here. I’m closing this PR and adding you as my co-authors in the new one. Feel free to contribute to the new PR.

**Pull request closed without merging**